java/client/src/test/java/glide/ExceptionHandlingTests.java

```diff
@@ -95,7 +94,7 @@ public void channel_is_closed_when_disconnected_on_command() {
         var channelHandler = new TestChannelHandler(callbackDispatcher);
         var commandManager = new CommandManager(channelHandler);
 
-        var future = commandManager.submitNewCommand(createDummyCommand(), r -> null);
```

Comment:
minor: I think that `createDummyCommand` conveyed intention rather nicely. consider `sendDummyCommand` instead.

Reply:
createDummyCommand made more sense when we had the Command.java, but that was removed in this PR.  Instead, we're just passing the object parameters directly. 
I DO agree that the "dummy" command conveyed information to a reader that this parameter is irrelevant to the test.  Maybe I can do something about that and make it more obvious...

Reply:
Changes to using TEST_ARGS.  I think it adds about the same as the createDummyCommand()